app/src/main/java/org/eyeseetea/malariacare/data/database/model/Option.java

```diff
@@ -57,15 +57,15 @@
     /**
      * Reference to parent answer (loaded lazily)
      */
-    Answer answer;
+    org.eyeseetea.malariacare.data.database.model.Answer answer;
```

Comment:
this change has had to do the IDE when moving database within data, is the same but changed it no problem